### ISSUE TITLE
Freature Request: Window mode being the default

### ISSUE THREAD

**Issue by Fuzzel11** [DrazharLn/pracx#25]
Is there any way to make it so opening in window mode is the default? I always play in window mode due to a weird bug where all animations play way to fast(to the point where you can even see the blades of a copter rotating) in full screen but are running normally in window mode.

**Comment by DrazharLn** [comment]
Could add a config option for that.

Do you get the fast animations without PRACX too?

**Comment by Fuzzel11** [comment]
Yes, just tested it with a fresh GOG installation. It's not like the option in the preferences that makes all units move "quickly" though. They do play movement/fight animations but very very fast.

Only thing I can think of is that because it's an old game, the high frequency of modern gpus makes problems with the part of the game that plays animations when the game is in exclusive full screen.

I also tried having just one monitor connected (usually 3) but no difference :/

My specs if it's any help:
Intel 6700k @ 4Ghz
Nvidia GTX 1070
Windows 10 Edu 64bit (17134.285)

**Comment by DrazharLn** [comment]
So, to be clear, you do or do not get the same bug with a fresh gog install?

I don't get this issue and I have a similar rig, so it's not just about
modern hardware being faster.

**Comment by Fuzzel11** [comment]
I do get the same bug with a fresh gog install

**Comment by Fuzzel11** [comment]
Hey there, I was setting up my new Pc and installing SMAC again, and I got to try it out in a few different configurations and the fast animations happened in all scenarios:

* Fresh GOG Installation,
* GoG with DirectDraw=0
* GoG with PRACX

I even tried out installing SMAC in a Windows XP virtual machine, but there was still the fast animation play in fullscreen. I couldn't try out window mode because PRACX seems to be incompatible with Windows XP (didn't expect it too work to be honest). So maybe this is something that is a general bug in SMAC? Not sure if it's still an issue of modern hardware.

**Comment by DrazharLn** [comment]
Try installing scient's unofficial patch before installing pracx. https://github.com/DrazharLn/scient-unofficial-smacx-patch/releases/

On XP, did you get an error about the installer being 64 bit (or not being 32 bit)? I recently fixed that and the v1.11 installer on the releases page was updated a couple of days ago.

I think the actual pracx binaries should work on XP, though I haven't tested that and might have set the wrong compiler flags. If the most recent installer doesn't work, you can just download the .dlls, and pracxpatch.exe put them in the same dir as terran.exe and run pracxpatch.exe once. You can also pick an icons image from here: https://github.com/DrazharLn/pracx/tree/master/resources my preferred one is Icons.pcx, that should also go in the same dir as terran.exe.

You can generally open installers with unzip, too, if they don't work, btw.

**Comment by Fuzzel11** [comment]
On my Windows 10 machine I have my installation in this order:
* gog
* scient 2.1
* yitzi 3.5 + 3.5c
* pracx

(also tried without yitzi) and the issue with the too fast animations still occurs.
I am actually not sure in what order to properly handle the patches because they all seem to overwrite terranx.exe and both scient and yitzi overwrite the same files such as alphax.txt

About my Windows XP VM:
I got there the same fresh gog installation + scient 2.1 (installer works and game launches)

The v1.11 Installer fails on execution with the error:
"<PathToInstaller>\PRACX.v1.11.exe is not a valid Win32 application."

To get around that I tried to simply place both .dll's and pracxpatch.exe in my installation folder and executing pracxpatch.exe, which does work, but when trying to launch the game terranx.exe encounters a problem and closes.

I also tried copying a working installation from my Win10 host to my XP VM but it has the same problem with terranx.exe encountering a problem.

So I think the patched executable does not work in XP. There is technical information about the error report I can view, but sadly there is no way to copy the content of it, because the text window of it is not selectable, but I did include the other file that it says that would be included in the report but I'm not sure how helpful that is: [55e3_appcompat.txt](https://github.com/DrazharLn/pracx/files/4691847/55e3_appcompat.txt)


![2020-05-27_22-57-13_VirtualBoxVM](https://user-images.githubusercontent.com/4890479/83072154-8ebfc280-a06e-11ea-9985-5bc9a5484898.png)
![2020-05-27_22-57-30_VirtualBoxVM](https://user-images.githubusercontent.com/4890479/83072155-8f585900-a06e-11ea-97be-dd3c1596f348.png)
![2020-05-27_23-07-57_VirtualBoxVM](https://user-images.githubusercontent.com/4890479/83072386-f83fd100-a06e-11ea-88ee-34cf23f23106.png)

**Comment by cmcaine** [comment]
You've been a bit unclear about whether these errors occur without PRACX, btw.

>  "\PRACX.v1.11.exe is not a valid Win32 application."

I think I fixed that yesterday. If you download the installer and try it again, it should work, but it almost certainly won't fix your other issues.

I don't really think I'm going to fix this animation bug, sorry. It would be very difficult to debug and I haven't touched the code in a while (and didn't fully understand it, even then).

I have no idea why it won't run on windows XP and I'm not going to spend time debugging it, sorry. I have had some luck playing SMACX with wine in linux, so you could try that, but, I'd stay clear of PRACX unless and until you get the base game working without crashing.

You can get basic higher def support by just setting DirectDraw=0 in the .ini

**Comment by DrazharLn** [comment]
You've been a bit unclear about whether these errors occur without PRACX, btw.

>  "\PRACX.v1.11.exe is not a valid Win32 application."

I think I fixed that yesterday. If you download the installer and try it again, it should work, but it almost certainly won't fix your other issues.

I don't really think I'm going to fix this animation bug, sorry. It would be very difficult to debug and I haven't touched the code in a while (and didn't fully understand it, even then).

I have no idea why it won't run on windows XP and I'm not going to spend time debugging it, sorry. I have had some luck playing SMACX with wine in linux, so you could try that, but, I'd stay clear of PRACX unless and until you get the base game working without crashing.

You can get basic higher def support by just setting DirectDraw=0 in the .ini
